### PR TITLE
Add pullquote citation

### PR DESCRIPTION
**Pull request by amberleyromo**
Adds a basic source citation to blog pullquotes, if supplied.

screen shot 2018-12-18 at 12 34 08 pm

### CODE REVIEW

File: www/src/components/shared/pullquote.js [hunk 1]
```diff
@@ -155,13 +163,14 @@ const Star = styled(`span`)`
...
 
-const Pullquote = ({ children }) => {
+const Pullquote = ({ citation, children }) => {
   instancesCounter += 1
   const className = `variant${variants[instancesCounter % variants.length]}`
 
   return (
     <PullquoteRoot className={className}>
       {children}
+      {citation && <Citation>- {citation}</Citation>}
```
Comment (DSchau) [inline www/src/components/shared/pullquote.js]: nit: but should this be an emdash or endash maybe?

```suggestion
       {citation && <Citation>&mdash;{citation}</Citation>}
```

**Comment by amberleyromo** [comment]
Hmm. @shannonbux do you have any opinion on convention for source citation? (see @DSchau's note above). 

Personally I think it's situational. It probably makes sense to have anything from the Gatsby team be noted as such.

File: www/src/components/shared/pullquote.js [hunk 2]
```diff
@@ -37,6 +37,14 @@ const PullquoteRoot = styled(`blockquote`)`
   }
 `
 
+const Citation = styled(`cite`)`
+  display: block;
+  text-align: right;
```
Comment (greglobinski) [inline www/src/components/shared/pullquote.js]: A nit. The alphabetical order of props, so `text-align` goes at the end. 

I also do not care in what order the properties like `font-weight` and `font-style` goes. Only the first character counts for me.

Comment (amberleyromo) [reply www/src/components/shared/pullquote.js]: Lol, done. Thanks for enforcing.

**Comment by shannonbux** [comment]
This looks great. I do think the company would be nice to know!

**Pull request merged**